descriptor.json

```diff
@@ -0,0 +1,29 @@
+{
+        "package": {
```

Comment:
Nit: Bad indentation. You can configure your editor to handle the beautification of the file by itself.